tests/unit/backend/corpora/api_server/curator/collection/test_dataset.py

```diff
@@ -114,5 +114,53 @@ def test___conflict_curator_tag(self):
         self.assertIsNone(dataset_2.curator_tag)
 
 
+class TestGetDatasets(BaseAuthAPITest):
+    def test_get_multiple_datasets_in_a_collection_200(self):
+        collection = self.generate_collection(self.session, visibility=CollectionVisibility.PRIVATE.name)
+        self.generate_dataset(self.session, collection=collection)
+        self.generate_dataset(self.session, collection=collection)
+        test_url = f"/curation/v1/collections/{collection.id}/datasets"
+        response = self.app.get(test_url)
+        self.assertEqual(200, response.status_code)
+        self.assertEqual(2, len(response.json["datasets"]))
+
+    def test_get_multiple_datasets_in_a_collection_ignore_tombstones_200(self):
+        collection = self.generate_collection(self.session, visibility=CollectionVisibility.PRIVATE.name)
+        self.generate_dataset(self.session, collection=collection)
+        self.generate_dataset(self.session, collection=collection, tombstone=True)
+        test_url = f"/curation/v1/collections/{collection.id}/datasets"
+        response = self.app.get(test_url)
+        self.assertEqual(200, response.status_code)
+        self.assertEqual(1, len(response.json["datasets"]))
+
+    def test_get_fake_dataset_404(self):
```

Comment:
```suggestion
    def test_get_nonexistent_dataset_404(self):
```
"fake" implies "stub" to me, esp in context of tests